lib/addons/adot/index.ts

```diff
@@ -0,0 +1,57 @@
+import { CoreAddOn, CoreAddOnProps } from "../core-addon";
+import { ClusterInfo } from "../../spi";
+import { getAdotCollectorPolicyDocument } from "./iam-policy";
+import { loadYaml, readYamlDocument } from "../../utils";
+import { KubernetesManifest } from "aws-cdk-lib/aws-eks";
+
+/**
+ * Configuration options for the Adot add-on.
+ */
+ export interface AdotCollectorAddOnProps extends CoreAddOnProps 
+
+const defaultProps = {
+    addOnName: 'adot',
+    saName: 'adot-collector',
+    policyDocumentProvider: getAdotCollectorPolicyDocument,
+    namespace: 'default'
```

Comment:
It does not seem right to install infrastructure components in the default namespace. Can we default to a dedicated namespace?